src/index/IndexImpl.cpp

```diff
@@ -265,10 +288,12 @@ std::pair<size_t, size_t> IndexImpl::createInternalPSOandPOS(
   auto onDiskBaseBackup = onDiskBase_;
   auto configurationJsonBackup = configurationJson_;
   onDiskBase_.append(QLEVER_INTERNAL_INDEX_INFIX);
-  auto internalTriplesUnique = ad_utility::uniqueBlockView(
-      internalTriplesPsoSorter.template getSortedBlocks<0>());
-  createPSOAndPOSImpl(NumColumnsIndexBuilding, std::move(internalTriplesUnique),
-                      false);
+
+  auto sortedBlockView{internalTriplesPsoSorter.template getSortedBlocks<0>()};
+  auto internalTriplesUnique =
+      ad_utility::uniqueBlockView(std::move(sortedBlockView));
```

Comment:
Was the splitting up here done to debug something? It should now make a difference, so either comment on what is going on here, or reinstate the old code.